aspnetcore/host-and-deploy/azure-iis-errors-reference.md

```diff
@@ -15,7 +15,7 @@ uid: host-and-deploy/azure-iis-errors-reference
 
 By [Luke Latham](https://github.com/guardrex)
 
-The following isn't a complete list of errors. Should an error occur that isn't listed here, please leave a detailed error message in the comments section below.
+The following isn't a complete list of errors. If you encounter an error not listed here, [open a new issue](https://github.com/aspnet/Docs/issues/new) with detailed instructions or [create a PR](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/azure-iis-errors-reference) for this doc.
```

Comment:
maybe ...

> ... with detailed instructions to reproduce the error ...

wrt creating the PR out-of-the-blue, I bet for perceived errors that we'll want to discuss it with them in an issue prior to having them go to that trouble. Some of these will end up being misconfigurations that don't apply to IIS/Antares. Just guessing tho.